### ISSUE TITLE
Checkpoints for Synthetic Burst SR does not match with the model

### ISSUE THREAD

**Issue by Magauiya** [akshaydudhane16/Burstormer#12]
Dear authors!

Thank you for sharing your work with the community. Recently, I tried to inference Burstormer on Synthetic Burst SR validation set and found that provided **checkpoint does not match with the model**:

Missing key(s) in state_dict: "back_projection1.feat_fusion.0.weight", "back_projection1.feat_fusion.0.bias", "back_projection1.feat_expand.0.weight", "back_projection1.feat_expand.0.bias", "back_projection2.feat_fusion.0.weight", "back_projection2.feat_fusion.0.bias", "back_projection2.feat_expand.0.weight", "back_projection2.feat_expand.0.bias". 
	Unexpected key(s) in state_dict: "back_projection1.diff_fusion.weight", "back_projection1.feat_fusion.weight", "back_projection1.feat_expand.weight", "back_projection2.diff_fusion.weight", "back_projection2.feat_fusion.weight", "back_projection2.feat_expand.weight". 
	size mismatch for align.alignment0.back_projection.encoder1.0.norm1.body.weight: copying a param with shape torch.Size([**96**]) from checkpoint, the shape in current model is torch.Size([**48**]).
	size mismatch for align.alignment0.back_projection.encoder1.0.norm1.body.bias: copying a param with shape torch.Size([**96**]) from checkpoint, the shape in current model is torch.Size([**48**]).
	size mismatch for align.alignment0.back_projection.encoder1.0.attn.qk.weight: copying a param with shape torch.Size([**192, 96, 1, 1**]) from checkpoint, the shape in current model is torch.Size([**96, 48, 1, 1**]).  

I increased the num. of channels twice in ref_back_projection.encoder1, but still there is a mismatch in no_ref_back_projection. Checkpoint contains **diff_fusion** block weights, but the Burstormer model does not have it. 

Can you please help to resolve this issue?

**Comment by DADAstrong** [comment]
> Dear authors!
> 
> Thank you for sharing your work with the community. Recently, I tried to inference Burstormer on Synthetic Burst SR validation set and found that provided **checkpoint does not match with the model**:
> 
> Missing key(s) in state_dict: "back_projection1.feat_fusion.0.weight", "back_projection1.feat_fusion.0.bias", "back_projection1.feat_expand.0.weight", "back_projection1.feat_expand.0.bias", "back_projection2.feat_fusion.0.weight", "back_projection2.feat_fusion.0.bias", "back_projection2.feat_expand.0.weight", "back_projection2.feat_expand.0.bias". Unexpected key(s) in state_dict: "back_projection1.diff_fusion.weight", "back_projection1.feat_fusion.weight", "back_projection1.feat_expand.weight", "back_projection2.diff_fusion.weight", "back_projection2.feat_fusion.weight", "back_projection2.feat_expand.weight". size mismatch for align.alignment0.back_projection.encoder1.0.norm1.body.weight: copying a param with shape torch.Size([**96**]) from checkpoint, the shape in current model is torch.Size([**48**]). size mismatch for align.alignment0.back_projection.encoder1.0.norm1.body.bias: copying a param with shape torch.Size([**96**]) from checkpoint, the shape in current model is torch.Size([**48**]). size mismatch for align.alignment0.back_projection.encoder1.0.attn.qk.weight: copying a param with shape torch.Size([**192, 96, 1, 1**]) from checkpoint, the shape in current model is torch.Size([**96, 48, 1, 1**]).
> 
> I increased the num. of channels twice in ref_back_projection.encoder1, but still there is a mismatch in no_ref_back_projection. Checkpoint contains **diff_fusion** block weights, but the Burstormer model does not have it.
> 
> Can you please help to resolve this issue?

Hi, did you solve this issue?

**Comment by YoungP2001** [comment]
Hello, I have the same question. Did you solve it？
